frontend/pages/2023.vue

```diff
@@ -61,7 +61,7 @@ const showVideo = ref(false)
                 <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24"><path fill="currentColor" d="M12 6.5A2.5 2.5 0 0 1 14.5 9a2.5 2.5 0 0 1-2.5 2.5A2.5 2.5 0 0 1 9.5 9A2.5 2.5 0 0 1 12 6.5M12 2a7 7 0 0 1 7 7c0 5.25-7 13-7 13S5 14.25 5 9a7 7 0 0 1 7-7m0 2a5 5 0 0 0-5 5c0 1 0 3 5 9.71C17 12 17 10 17 9a5 5 0 0 0-5-5Z"/></svg>
                 <span>Kode Creative Hub, Depok Town Square<br/>Depok, Indonesia</span>
               </h3>
-              <Btn :is-link="true" to="/" class="mt-5 lg:mt-15 text-center btn-save-my-spot" size="xl">Back to Current Event</Btn>
+              <Btn v-umami="{name: 'go-to-2024-page-btn'}" :is-link="true" to="/" class="mt-5 lg:mt-15 text-center btn-save-my-spot" size="xl">Back to Current Event</Btn>
```

Comment:
If we're moving to Nuxt's useHead composable, we should be using the custom umami directive. That should be easy enough.